lib/chromedriver.js

```diff
@@ -395,15 +395,18 @@ class Chromedriver extends events.EventEmitter {
         await this.proc.stop();
       }
 
+      let message = '';
       // often the user's Chrome version is too low for the version of Chromedriver
       if (e.message.indexOf('Chrome version must be') !== -1) {
-        log.error('Unable to automate Chrome version because it is too old for this version of Chromedriver.');
+        message = message.concat('Unable to automate Chrome version because it is too old for this version of Chromedriver.\n');
         if (webviewVersion) {
-          log.error(`Chrome version on device: ${webviewVersion}`);
+          message = message.concat(`Chrome version on device: ${webviewVersion}\n`);
         }
-        log.error(`Please see 'https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/web/chromedriver.md'`);
+        message = message.concat(`Please see 'https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/web/chromedriver.md'\n`);
       }
-      log.errorAndThrow(e);
+
+      message = message.concat(e);
```

Comment:
I've updated _after_ part in this description.
It looks enough for `e.message` since the difference is the server-side stack trace. For users, the link to documentation and versions of chrome browser and the driver. 👍